scripts/historical_treasury_exporter.py

```diff
@@ -1,22 +1,31 @@
 import logging
 from datetime import datetime, timezone
 
+from brownie import chain
 from yearn.historical_helper import export_historical, time_tracking
+from yearn.networks import Network
 from yearn.treasury.treasury import YearnTreasury
 from yearn.utils import closest_block_after_timestamp
 
 logger = logging.getLogger('yearn.historical_treasury_exporter')
 
 def main():
     start = datetime.now(tz=timezone.utc)
-    # end: 2020-07-21 first treasury tx
-    end = datetime(2020, 7, 21, 10, 1, tzinfo=timezone.utc)
+    if Network(chain.id) == Network.Fantom:
```

Comment:
same two comments apply here

Reply:
fix'd, commit pending

Reply:
I used dict(details)[chain.id] syntax instead of dict(details).get(chain.id) syntax to show we require chainid to be supported